hashicorp/web-unified-docs
# Chore: Remove Packer content

# Changes

- Remove packer content from code base to prep for education engineer test. This includes:
  - Removing packer from the `content` dir
  - Removing packer from the `public/assets` dir

- [Asana task](https://app.asana.com/0/1207899860738460/1208562243516324/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
```
curl https://web-presence-experimental-docs.vercel.app/api/content/packer/nav-data/latest/docs
```
1. You should receive a valid response, such as this:
<img width="1147" alt="Screenshot 2024-10-16 at 16 15 11" src="https://github.com/user-attachments/assets/1f8b3a08-b2d4-4c66-b7a6-b2f5ac9ba101">


1. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-0e9e3d-hashicorp.vercel.app/api/content/packer/nav-data/latest/docs
```
1. You should receive a response of `Not Found`
